<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeWordSentimentDialog</class>
 <widget class="QDialog" name="changeWordSentimentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить тональность слова</string>
  </property>
  <widget class="QLineEdit" name="wordEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите слово</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelWord">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Слово:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="chooseSentimentGroupBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>191</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Тональность</string>
   </property>
   <widget class="QRadioButton" name="radioButtonPSTV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Положительная</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNGTV">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Отрицательная</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonNEUT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Нейтральная</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChange">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
